internal/plugin/provider.go

```diff
@@ -114,16 +113,8 @@ func (p *AivenProvider) Resources(context.Context) []func() resource.Resource {
 		organization.NewOrganizationResource,
 		organization.NewOrganizationUserGroupMembersResource,
 		organization.NewOrganizationGroupProjectResource,
-	}
-
-	// Add to a list of resources that are currently in beta.
-	if util.IsBeta() {
-		betaResources := []func() resource.Resource{
```

Comment:
Please don't remove it and make it an empty slice instead. We should make it easy to add new beta resources when we need it in future.

Reply:
I would like to make it a different way instead.

Reply:
The problem is that the current implementation of the incremental acceptance testing solution relies on the ability to be able to statically parse the file.

The only way to make it is to generate a file that's similar to this at the end. I'm afraid, dynamic solutions like loops over description builder won't work, sadly 🙁